_config.yml

```diff
@@ -92,47 +94,45 @@ paginate_path: "blog/page:num"
 markdown: kramdown
 kramdown:
   parse_block_html: true
-permalink: /blog/:year/:month/:title/
-highlight: rouge
-plugins_dir: [jekyll-paginate, jekyll-redirect-from]
+permalink: /blog/:year/:month/:title.html
+highlight: pygments
 excerpt_separator: "<!--more-->"
-include: ['.htaccess']
-# Exclude these files from your production _site
+include:
+  - .htaccess
 exclude:
-    - .sass-cache
-    - Gemfile
-    - Gemfile.lock
-    - LICENSE
-    - README.md
-    - INSTALL.md
-    - vendor/
+  - .sass-cache
+  - Gemfile
+  - Gemfile.lock
+  - LICENSE
+  - README.md
+  - INSTALL.md
+  - Makefile
+  - vendor/
+
+# Build settings
+# - https://github.com/jekyll/jekyll-redirect-from
 
 plugins:
-  - jekyll-get-json
+  - jekyll-paginate
+  - jekyll-redirect-from
+   -jekyll-get-json
 
 ## external data sources
 jekyll_get_json:
   - data: all_badged_people
     json: 'https://feeds.carpentries.org/all_badged_people.json'
-  - data: all_past_workshops
-    json: 'https://feeds.carpentries.org/all_past_workshops.json'
-  - data: all_upcoming_workshops
-    json: 'https://feeds.carpentries.org/all_upcoming_workshops.json'
   - data: badges_stats
     json: 'https://feeds.carpentries.org/badges_stats.json'
-#  - data: community_lessons
-#    json: 'https://feeds.carpentries.org/community_lessons.json'
+  - data: community_lessons
+    json: 'https://feeds.carpentries.org/community_lessons.json'
   - data: help_wanted_issues
     json: 'https://feeds.carpentries.org/help_wanted_issues.json'
-  - data: newsletter
-    json: 'https://feeds.carpentries.org/newsletter.json'
-
 
 # The language setting is used in /includes/header.html for html-settings
 language: "en"
 
 # Use an incremental build (if supported, see https://jekyllrb.com/docs/configuration/incremental-regeneration/)
-incremental: true
+incremental: false
```

Comment:
The build wasn't reflecting changes locally with this enabled, and I'm not sure the CI supports it. I could go either way on changing it back.

Reply:
Ok, I was curious because GitHub also complained about this recently which was why I moved the setting in here. I have no problem with leaving it `false`